### ISSUE TITLE
Failed to handle request textDocument/didChange

### ISSUE THREAD

**Issue by paulloz** [ephread/ink#1]
Win10 Pro, VSCode 1.45.1, no WSL shenanigans.

[vscode-ink](https://github.com/ephread/vscode-ink) is configured as so:
```json
{
    "ink.languageServer.inklecatePath": "path\\to\\inklecate.exe",
    "ink.languageServer.useSpecificRuntime": "none",
    "ink.useLanguageServer": true,
}
``` 

Whenever I save my `.ink` file and a call to the server is triggered, the following error raises:
```
Failed to handle request textDocument/didChange - System.Exception: The specified document is outside of the current workspace.
```

Full log file: [language_server20200518_001.txt](https://github.com/ephread/ink/files/4646973/language_server20200518_001.txt)

**Comment by ephread** [comment]
Thanks for reporting the issue @paulloz, after reviewing the logs I think the problem stems from how vscode encodes file URIs. `c:` is sent `c%3A` and is causing issues. It seems to be a well known problem:

- https://github.com/OmniSharp/csharp-language-server-protocol/issues/79
- https://github.com/microsoft/vscode-languageserver-node/issues/105
- https://github.com/microsoft/vscode/issues/58985

I'll see how I can handle that escaped colon.

**Comment by ephread** [comment]
Should be fixed in `language-server/HEAD`, let me how it goes!

**Comment by paulloz** [comment]
Yep, everything works as expected for me on d7ec1c1daacafdcb83a8de0c2a12bbacb4cc9b53.
